pages/journal/index.js

```diff
@@ -0,0 +1,88 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+function Journal({ journalData, journals }) {
```

Comment:
In order to distinguish this page from a component:
```suggestion
function JournalPage({ journalData, journals }) {
```